Address Dev & SME review

meswan opened the pull request:
Integrate with quicklabs

Commit:
update readme

Commit:
update readme

Commit:
Merge remote-tracking branch 'origin/dev' into dev

# Conflicts:
#	README.adoc

Commit:
update readme

Commit:
update readme

Commit:
update readme

Commit:
screenshot for server name

Commit:
match /finish file

Commit:
update commands for CICD

Add test script and gh action

Commit:
change to branch qa

Commit:
Add quicklabs draft converter

Commit:
trigger from draft branch

Commit:
Merge branch 'draft' into dev

Commit:
trigger conversion workflow

Commit:
Merge branch 'dev' of https://github.com/OpenLiberty/draft-guide-contract-testing into dev

Commit:
address dev content review

Commit:
address sme review

Commit:
address sme review

Commit:
address sme review

Commit:
address sme review

Commit:
address sme review

Commit:
address sme review

gkwan-ibm reviewed (changes requested):
@meswan 
- [x] the text in the picture at the "Starting pact broker" subsection looks small 
- [x] In the "Contract testing" subsection, "...CI/CD pipelines.", spelling out what are CI and CD, then the rest of the guide can simply use CI/CD. 
- **Verifying the contract**
  - [x] "Replace the SystemResource.java file." -> "Replace the SystemResource class."
  - [x] better reword the statement "With the `/version` endpoint corrected to return a decimal type, press the enter/return key..."  like
    - "Correct the `system.properties.version[hotspot]` value to decimal. Press the enter/return key..."
  - SystemResource.java
    - [x] why `getVersion()` return JsonArray? It only returns a value
    - [x] format the lines better 
      - ![Screen Shot 2021-01-12 at 10 54 15 AM](https://user-images.githubusercontent.com/29576299/104338817-e8bdad00-54c4-11eb-8633-c6dac3887df7.png)
    - [x] the indents looks too long. better use 2 or 4 spaces
  - [x]  the text in the picture at the "Verifying the contract" section looks small 
    - maybe all pictures. Is it possible to reduce the spacing between the columns (make the pictures less wide), so the browser may show the pictures bigger?

Commit:
address dev content feedback

reduce indent to two spaces
replace screenshots
Switch from jsonArray to JsonObject for getVersion

Commit:
Fix formatting

gkwan-ibm commented:
@meswan 
- [ ] should be consistent to use "...class." or "... class file." in the command boxes
- [ ] **Verifying the contract**
  - [ ] improve the order for "Correct the system.properties.version value to decimal and press the enter/return..." and the "Replace" command, either
   - "Replace" command, then "Correct the system.properties.version value to decimal and press the enter/return..."
   - "Correct the system.properties.version value to decimal",  "Replace" command, then "Press the enter/return..."
 - [ ] The SystemResource.java in the right pane, there are some unnecessary space lines

Commit:
address dev content review
